restli-tools/src/main/java/com/linkedin/restli/tools/data/ExtensionSchemaValidationCmdLineApp.java

```diff
@@ -53,9 +58,13 @@
 /**
  * This class is used to validate extension schema, the validation covers following parts:
```

Comment:
Rephrasing suggestions for clarity:
1. I don't understand what this means. What is a `validate schema`? Is this supposed to say `valid schema`?
2. Makes sense. (nit: replace hyphen `-` with colon `:`)
3. I would use a definite article like `can only include the base schema` to imply that there is only one base schema for an extension.
4. What does it mean to be annotation by a namespace? Does this mean to say all field annotation keys must be in the `extension` namespace? (nit: be careful of the distinction between "should" and "must")
5. It would be more accurate to say that the annotations "must conform to `{@link ExtensionSchemaAnnotation}`" (nit: be careful of the distinction between "should" and "must")
6. Makes sense.
7. Similar comment as in bullet 4.
8. Clarify which `resourceKey` annotation. Corresponding to what? (nit: remove the hyphen `-`)

Also, in the text at the top, nit: `to validate extension schema` -> `to validate extension schemas`

Reply:
Fixed